### PR TITLE
Rephrased the test cases as it was confusing

### PR DESCRIPTION
**Pull request by tisutisu**
During LPINTEROP testing observed that the test cases are bit confusing. So rephrased the test cases. Please review.

### CODE REVIEW

**Comment by nemesis09** [comment]
@tisutisu could you please attach a bug to the PR so that it can be tracked?

**Comment by sanketpathak** [comment]
@tisutisu You need to create a Bugzilla and mention it in the heading like [this](https://github.com/openshift/console/pull/10129)

**Comment by sanketpathak** [comment]
@makambalaji Need your approval

**Comment by sanketpathak** [comment]
@tisutisu Can you update the step definition file as well, you need to add the following code below this scenario https://github.com/openshift/console/blob/1bd4607153c8031262f6db5d6a61b20fe4292ce3/frontend/packages/helm-plugin/integration-tests/support/step-definitions/helm/helm-navigation.ts#L122-L124
```
When('user selects all the checkboxes {string}, {string} and {string}', (status1: string, status2: string, status3: string) => {
  helmPage.selectHelmFilter(status1);
  helmPage.selectHelmFilter(status2);
  helmPage.selectHelmFilter(status3);
});

Then('all the Helm charts are listed', () => {
  helmPage.verifyHelmChartsListed();
});
```
and update the line https://github.com/openshift/console/blob/1bd4607153c8031262f6db5d6a61b20fe4292ce3/frontend/packages/helm-plugin/integration-tests/support/step-definitions/helm/helm-navigation.ts#L138
to 
```
Then(`all the filters selected will get removed`, (status: string) => {
```

File: frontend/packages/helm-plugin/integration-tests/features/helm/helm-navigation.feature [hunk 1]
```diff
@@ -85,17 +85,17 @@ Feature: Navigations on Helm Chart
...
 
         @regression
         Scenario: Clear all filters: HR-05-TC10
             Given user is at the Helm page
              When user clicks on the filter drop down
-              And user selects checkbox for the "All" Helm charts
+              And user selects all the checkboxes "Deployed", "Failed" and "Others"
               And user clicks on the clear all filters button
-             Then "All" filters selected will get removed
+             Then All the filters selected will get removed
```
Comment (sanketpathak) [inline frontend/packages/helm-plugin/integration-tests/features/helm/helm-navigation.feature:98]:
```suggestion
             Then all the filters selected will get removed
```

**Pull request closed without merging**